oam/install_scripts/columnstore-post-install.in

```diff
@@ -174,36 +204,39 @@ mkdir $lockdir >/dev/null 2>&1
 
 rm -f $lockdir/columnstore
 
-#check and get amazon env variables
-aws=`which aws 2>/dev/null`
-if [ -z "aws" ]; then
-    MCSgetCredentials.sh >/dev/null 2>&1
+# This was the last place of postConfigure. RIP
+
+/usr/sbin/install_mcs_mysql.sh --tmpdir=$tmpDir
+
+# Restart MDB to enable plugin
+systemctl cat mariadb.service > /dev/null 2>&1
+if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
+    systemctl restart mariadb.service > /dev/null 2>&1
+else
+    pkill mysqld > /dev/null 2>&1
+    /usr/bin/mysqld_safe &
 fi
 
-postConfigure
+checkForError
+if [ $? -ne 0 ]; then
+    echo "ERROR: Invalid password in .my.cnf, or Columnstore plugin install missing"
+fi
 
 systemctl cat mariadb-columnstore.service > /dev/null 2>&1
 if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
     systemctl start mariadb-columnstore
-     
-    # Wait for all columnstore to be ready, DDLProc is final process in startup order
-    while  [ -z  "$(pgrep -x DDLProc)" ];
-    do
-        sleep 1
-    done
+    dbbuilder 7 > $tmpDir/dbbuilder.log
```

Comment:
i'm assuming with the change to mariadb-columnstore systemd unit, it will be up and running before ddbuilder is called?

Reply:
Yes. systemctl start/stop mariadb-columnstore are synchronous commands now.